pom.xml

```diff
@@ -93,19 +93,48 @@
         <version>4924.v6b_eb_a_79a_d9d0</version>
         <type>pom</type>
         <scope>import</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>trilead-api</artifactId>
+          </exclusion>
+        </exclusions>
       </dependency>
     </dependencies>
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>eddsa-api</artifactId>
```

Comment:
Could we remove this dependency and use this new feature from mina-sshd see https://github.com/apache/mina-sshd/pull/639
The project https://github.com/str4d/ed25519-java is not maintained anymore and have somen know security issues such https://github.com/str4d/ed25519-java/issues/95